resources/js/components/Create.vue

```diff
@@ -88,8 +98,19 @@
                 this.idForTodoTitle='';
                 this.newTodoBody = '';
                 this.idForTodoBody='';
-            }
+            },
 
+            remove(id){
+                let confirmed = confirm('Are you sure you want to delete?');
+                if (confirm){
+                    /**JavaScript**/
+                    // console.log(id);
+                    // this.todos.splice(id,1)
+
+                    /**VueJS**/
+                    this.$delete(this.todos, id);
```

Comment:
This is using the VueJS default delete method which is different to the actions/mutators we define in the store.
You should use the store dispatch method here, calling the delete action you defined.